libs/qrencode/Makefile

```diff
@@ -40,6 +42,14 @@ as a mobile phone with CCD. The capacity of QR Code is up to 7000
 digits or 4000 characters, and is highly robust.
 endef
 
+define Package/libqrencode/config
+  config QRENCODE_PNG
+	bool "Enable PNG output"
+	depends on PACKAGE_libqrencode
+	select PACKAGE_libpng
```

Comment:
this should be done via `DEPENDS` already

Reply:
Yes, but in that case the dependency will not be visible in menuconfig Help.